### PR TITLE
Added an option to debug the map using shp2img

### PR DESCRIPTION
**Pull request by samuellapointe**
This is the first step in my project to debug the scribe files in ScribeUI. Right now, it only displays the raw output of shp2img. The next step will be getting a clearer error output, accurate to the scribe files, using this debug output.

### CODE REVIEW

File: scribe.py [hunk 1]
```diff
@@ -642,6 +651,8 @@ def main():
                 
             try:
                 jsonToMap(jsonContent, outputDirectory, mapName, clean)
+                if(debugLevel >= 0 and debugLevel <= 5): #debug using shp2img
```
Comment (cbourget) [inline scribe.py]: Please remove the parenthesis in the if.

**Comment by cbourget** [comment]
Works well. Please add a delimiter in the output, before the MS logs, indicating that these are the MS logs. Something like :
```
MapServer logs (log level 5)
----------------------------
```

**Comment by samuellapointe** [comment]
Changes applied!

**Comment by cbourget** [comment]
Thank you.

**Pull request merged**